docs/examples.rst

```diff
@@ -677,6 +677,34 @@ variables or context dimensions:
         ref: $@distro
 
 
+.. _anchors-aliases:
+
+Using anchors and aliases in Plans
+------------------------------------------------------------------
+
+When you need to specify the same variable multiple times,
+YAML has a feature called anchors and aliases. You can define an
```

Comment:
Could you try to find it in YAML specification and link it? To save users googling.

Reply:
The question is do I use https://pyyaml.org/wiki/PyYAMLDocumentation which doesn't have any links to specific headings, but is way more descriptive
Or do I use the https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases which is just theoretical